docs/user-guide/aroma/installing-payloadloader.md

```diff
@@ -15,7 +15,8 @@ Installing the PayloadLoader will let you access Aroma just by running the Healt
 1. Take the SD Card out of your PC and plug it into your Wii U.
 1. Execute the web browser exploit as [previously explained](browser-exploit) but this time, you will need to hold the X button to open the Environment Loader menu.
 1. Press A to launch the `aroma` environment.
-    - You will get a warning message about the update folder existing. This is normal. Press X to block updates.
+   
+?> You will get a warning message about the update folder existing. **This is normal.** Press X to block updates.
```

Comment:
Couldn't we have left it how it was before and just bold the `This is normal.`?

Reply:
I guess we can. Just wanted to prevent users from panicking and asking "what do I do now"